sonic-xcvrd/xcvrd/sff_mgr.py

```diff
@@ -435,6 +435,12 @@ def task_worker(self):
                     # Skip if these essential routines are not available
                     continue
 
+                try:
+                    api = sfp.get_xcvr_api()
```

Comment:
`api` has been already obtained at the code before: 
https://github.com/sonic-net/sonic-platform-daemons/blob/78d427e2659089fb80bc60e1b2db63fe917489fc/sonic-xcvrd/xcvrd/sff_mgr.py#L410